### ISSUE TITLE
Feature Request: ignore plain text (non-"shortcut") keystrokes

### ISSUE THREAD

**Issue by amynbe** [NickeManarin/ScreenToGif#471]
When capturing a gif in a text editor, it makes more sense to only show key strokes that don't appear directly in the text editor.
In other words, If I type a 1000-character text in my editor, I don't want the full text to appear in subtitles but only key strokes like:
- tab
- ctrl+v
- enter

**Comment by NickeManarin** [comment]
Nice request. :)

**Comment by WELZ-gh** [comment]
Good idea, but there should be an option when applying keystrokes to ignore "plain text" which can be enabled/disabled. 

There are programs (such as Illustrator or Photoshop) where regular keys toggle different tools etc.

**Comment by amynbe** [comment]
But Illustrator and photoshop are not text editors, right? In those cases it would be easy to not use this feature and just delete your plain text from the existing key strokes dialog. That would work because there shouldn't be a lot of plain text, as opposed to the potential large amount of plain text that you would input in a text editor.

**Comment by b-a0** [comment]
Looking for the same thing, would be a very nice addition!  
Optionally this could be part of the key stroke editor where you could have a drop-down filter to select what type of key strokes (e.g. single characters, chords with 2 keys, chords with 3 keys, long presses, etc...) should be taken into account in the Key Strokes overlay.
